entrypoint.R

```diff
@@ -7,6 +7,9 @@ use_archive <- Sys.getenv("CRANIUM_USE_ARCHIVE")
 use_hardlinks <- Sys.getenv("CRANIUM_USE_HARDLINKS")
 latest_only <- Sys.getenv("CRANIUM_LATEST_ONLY")
 fields <- Sys.getenv("CRANIUM_REPO_FIELDS")
+host_addr <- Sys.getenv("CRANIUM_HOST_ADDRESS", "0.0.0.0")
+host_port <- Sys.getenv("CRANIUM_PORT", 8000)
```

Comment:
Do you think there's any advantage to being able to modify the internal port? I know that some containers do this, but it seems pointless to me.

Reply:
Yeah, the reason for this is to easily test outside of containers. Inside the container we're not worried, but we write it all as though it might not be inside the container

Reply:
This has come up on some internal stuff. Being able to modify the port is quite handy on multi-user systems when you want to debug code outside of the container and you're not necessarily the only person debugging on that system. I'm thinking this is generally valuable.